pkg/plugin/config.go

```diff
@@ -110,5 +114,21 @@ func (cfg *PluginConfig) ToFlags(set *cli.FlagSet) *cli.FlagSet {
 		Usage:   "The resource name of [google.cloud.secretmanager.v1.SecretVersion].",
 	})
 
+	f.StringVar(&cli.StringVar{
```

Comment:
I think we need to either give them default value or validate they are not empty. 

Here is an example of specifying default value: https://github.com/abcxyz/jvs/blob/main/pkg/config/justification_config.go#L106

Reply:
I updated with default value.

Reply:
I'm not sure these make sense as default values. Default values should work "as-is" without customization. I don't think that's the default here.

Reply:
Agree that Hint default value "Jira Issue Key under specific project" is awkward here and we should require input and check not-empty. 

DisplayName default value "Jira Issue Key" seems reasonable? I don't have strong opinion on requiring a value

Reply:
I am no sure if I understand correctly.
If the input is empty, then should we throw an error as: https://github.com/abcxyz/jvs-plugin-jira/blob/main/pkg/plugin/config.go#L57-L60? 

And @yolocs What do you think?

Reply:
I think that's what @dandans-verily meant by:

> validate they are not empty.

Usually this is done in `Run` with something like if `c.flag == "" { return fmt.Errorf("missing required flag -flag") }`

Reply:
For hint, we might desire something more specific. Like "... under FOO project. Example: ..." And FOO would depend on your choice.

Reply:
I keep the DisplayName with default value and update the Hint with not-empty check.